barchart/charts/barChart.html

```diff
@@ -1,8 +1,10 @@
-<div ng-controller='barChartCtrl' layout="row" layout-align="center center">
+<div ng-controller='barChartCtrl'
+     layout="row"
+     layout-align="center center">
     <!-- pass max from form -->
     <bar-chart 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/barchart/charts/barChart.html
+++ b/barchart/charts/barChart.html
@@ -2,7 +2,7 @@
      layout="row"
      layout-align="center center">
     <!-- pass max from form -->
-    <bar-chart 
+    <bar-chart
         data="$storage.hashtagDateFreq"
         max="query.maxHashtags"
         title="Top Hashtags by day" >
```